src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisation.java

```diff
@@ -98,6 +100,48 @@ private String buildTextMessageDetails(Subscription subscription) {
         return buildTextMessage.toString();
     }
 
+    public Map<String, String> setAppointeeName(Map<String, String> personalisation, SscsCaseData sscsCaseData) {
+        Appointee appointee = sscsCaseData.getAppeal().getAppellant().getAppointee();
+        if (isValidAppointee(appointee)) {
+            personalisation.put(AppConstants.APPOINTEE_NAME, String.format("%s %s",
+                appointee.getName().getFirstName(),
+                appointee.getName().getLastName()));
+        }
+        return personalisation;
+    }
+
+    private boolean isValidAppointee(Appointee appointee) {
+        return null != (appointee) && null != appointee.getName();
+    }
+
+    public Map<String, String> setAppointeeDetails(Map<String, String> personalisation, SscsCaseData ccdResponse) {
+        personalisation.put(AppConstants.APPOINTEE_DETAILS_LITERAL, buildAppointeeDetails(ccdResponse.getAppeal().getAppellant().getAppointee()));
+        return personalisation;
+    }
+
+    private String buildAppointeeDetails(Appointee appointee) {
+        String hasAppointee = (appointee != null) ? YES : NO;
+
+        StringBuilder appointeeBuilder = new StringBuilder()
+            .append("Have a appointee: ")
+            .append(hasAppointee);
+
+        if (appointee != null) {
+            appointeeBuilder.append(TWO_NEW_LINES + "Name: ")
+                .append(appointee.getName().getFullNameNoTitle() + TWO_NEW_LINES)
```

Comment:
Are null checks needed for the train wreck?

Reply:
Yes its require if appointee is null then it will throw error at appointee.getName()

Reply:
The train here is appointee.getName().getFullNameNoTitle()